### PR TITLE
Replace Quick Start branding; replace quickstart attributes

### PR DESCRIPTION
**Pull request by DelfinGala**
* Replace "Quick Start" with "Partner Solution"
* Replace "quickstart" attributes with "partner-solution"
* Update a few outlying "http" mentions to "https"

### CODE REVIEW

**Review by annaone — DISMISSED** [review]

File: deployment_cdk/.specific/_settings.adoc [hunk 1]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline deployment_cdk/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: deployment_cfn/.specific/_settings.adoc [hunk 2]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline deployment_cfn/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: deployment_ct/.specific/_settings.adoc [hunk 3]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline deployment_ct/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: deployment_eks/.specific/_settings.adoc [hunk 4]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline deployment_eks/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: deployment_terraform/.specific/_settings.adoc [hunk 5]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline deployment_terraform/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: migration/.specific/_settings.adoc [hunk 6]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline migration/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: operational/.specific/_settings.adoc [hunk 7]
```diff
@@ -1,5 +1,5 @@
-:quickstart-project-name: quickstart-repo-name
-:quickstart-github-org: aws-quickstart
+:partner-solution-project-name: partner-solution-repo-name
+:partner-solution-github-org: aws-partner-solution
```
Comment (dowen12) [inline operational/.specific/_settings.adoc:2]: The 2nd part needs to remain as aws-quickstart at the aws-partner-solution org doesn't exist:
```suggestion
:partner-solution-github-org: aws-quickstart
```

File: introduction_operational.adoc [hunk 8]
```diff
@@ -1,8 +1,8 @@
 [.text-center]
 [discrete]
-== Quick Start Operational Guide
+== Partner Solution Operational Guide
 
-// Do not change the URL below. The aws-quickstart-graphic.png icon needs to come from the aws-quickstart S3 bucket.
+// Do not change the URL below. The aws-partner-solution-graphic.png icon needs to come from the aws-partner-solution S3 bucket.
```
Comment (dowen12) [inline introduction_operational.adoc:5]: this should still reference the aws-quickstart bucket
```suggestion
// Do not change the URL below. The aws-partner-solution-graphic.png icon needs to come from the aws-quickstart S3 bucket.
```

File: introduction.adoc [hunk 9]
```diff
@@ -1,8 +1,8 @@
 [.text-center]
 [discrete]
-== Quick Start Deployment Guide
+== Partner Solution Deployment Guide
 
-// Do not change the URL below. The aws-quickstart-graphic.png icon needs to come from the aws-quickstart S3 bucket.
+// Do not change the URL below. The aws-partner-solution-graphic.png icon needs to come from the aws-partner-solution S3 bucket.
```
Comment (dowen12) [inline introduction.adoc:5]: this should still reference the aws-quickstart bucket
```suggestion
// Do not change the URL below. The aws-partner-solution-graphic.png icon needs to come from the aws-quickstart S3 bucket.
```

File: introduction_migration.adoc [hunk 10]
```diff
@@ -1,8 +1,8 @@
 [.text-center]
 [discrete]
-== Quick Start Migration Guide
+== Partner Solution Migration Guide
 
-// Do not change the URL below. The aws-quickstart-graphic.png icon needs to come from the aws-quickstart S3 bucket.
+// Do not change the URL below. The aws-partner-solution-graphic.png icon needs to come from the aws-partner-solution S3 bucket.
```
Comment (dowen12) [inline introduction_migration.adoc:5]: this should still reference the aws-quickstart bucket
```suggestion
// Do not change the URL below. The aws-partner-solution-graphic.png icon needs to come from the aws-quickstart S3 bucket.
```

**Review by dowen12 — CHANGES_REQUESTED** [review]
Some changes requested inline, with suggestions

**Review by dowen12 — APPROVED** [review]
Happy with the changes

**Pull request merged**